### ISSUE TITLE
Source codes file from official site's checksum changed

### ISSUE THREAD

**Issue by aisk** [racket/racket#3639]
The minimal-racket source codes from https://download.racket-lang.org/all-versions.html of version 7.9's sha256 should be `293aa8ef709a6240472f16833351ba66a9e461261d2813c7fb1cc5ddf59c3000`, because homebrew have record it here: https://github.com/Homebrew/homebrew-core/blob/master/Formula/minimal-racket.rb#L5 .

But now it changed to `85b201aebc1ad1ec98ac590e18052d7ef8a81af280244d00ca1c28e8543b3fe9`, without any version changed or any release notes. Is there any changes after 7.9' releases?

I created a PR for homebrew: https://github.com/Homebrew/homebrew-core/blob/master/Formula/minimal-racket.rb#L5 , but seems they need explanation for this, to avoid any malicious circumstances.

**Comment by jbclements** [comment]
Yes, the source bundles were updated, and also the SHA1s on the web site.

See this discussion to understand the change:

https://groups.google.com/g/racket-users/c/aOJPcvoJrk8/m/WEVqAFlmAgAJ?pli=1

I can’t duplicate the sha256’s you report; shasum gives me a different string from either of the two you report on:

```
hardy:/tmp clements> shasum -a 256 < racket-minimal-7.9-src.tgz     
2ab2b28cf77a58a518f6b73d57cc6d154a570da7f41e0389915bad80effc8579  -
```

nevertheless, it seems reasonable to suggest that homebrew computes sha256’s on a slightly different sequence of bytes, so the short answer is: yes, the source bundle was updated.

John Clements

**Comment by shhyou** [comment]
@jbclements I think aisk is using the Source + built packages variant here.

**Comment by SMillerDev** [comment]
Next time, for package managers sakes, please consider a 3.9.1 version instead of you patch some part of the release.
The [git manual](https://git-scm.com/docs/git-tag#_on_re_tagging) says re-tagging is "the insane thing" to do.

**Comment by samth** [comment]
@SMillerDev note that this was only a packaging issue, and the tags were not changed.

**Comment by sorawee** [comment]
No longer an issue, and it's 8.0 now anyway.
